docs/crio.conf.5.md

```diff
@@ -239,6 +239,15 @@ The `crio.network` table containers settings pertaining to the management of CNI
 **plugin_dirs**=["/opt/cni/bin/",]
   List of paths to directories where CNI plugin binaries are located.
 
+## CRIO.METRICS TABLE
+The `crio.metrics` table containers settings pertaining to the Prometheus based metrics retrieval.
+
+**enable_metrics**=false
+  Globally enable or disable metrics support.
+
+**metrics_port**="9090"
```

Comment:
if we leave it an int this needs to change too

Reply:
Changed as suggested.